mockbee-test/ecom-store/src/components/Navbar/Navbar.jsx

```diff
@@ -1,6 +1,7 @@
 import { Link } from "react-router-dom";
-
+import { useFilter } from "../../context/filter-product-context";
```

Comment:
Same as above, have an index.js file, so when you import you only have to give path till /context